templates/inactive_reviewer_needinfo.txt

```diff
@@ -0,0 +1,19 @@
+The following {{ plural('patch is', revisions, 'patches are') }} waiting for review from an inactive reviewer:
+
+| ID  | Title |  Author | Inactive Reviewer |
+| --- | :---- |  ------ | ----------------- |
+{% for rev in revisions -%}
+{% for reviewer in rev['reviewers'] -%}
+{% if rev['reviewers'][0] == reviewer -%}
+| [D{{ rev['rev_id'] }}](https://phabricator.services.mozilla.com/D{{ rev['rev_id'] }}) |  {{ rev['title'] | e }} | [{{ rev['author']['nick'] }}](https://bugzilla.mozilla.org/user_profile?user_id={{ rev['author']['id'] }})  |
+{%- else -%}
+| | | |
+{%- endif -%}
+
+[{{ reviewer['phab_username'] }}](https://phabricator.services.mozilla.com/p/{{ reviewer['phab_username'] }}/) |
+{% endfor -%}
+{% endfor %}
+
+:{{ nicknames }}, could you please find an active reviewer?
```

Comment:
```suggestion
:{{ nicknames }}, could you please find another reviewer or abandon the patch if it is no longer relevant?
```

Could we add "or abandon the patch if it is no longer relevant" in case the patch is old?

Reply:
Done in 201c2f3eb24e6c0b27fc3d7565ab5bb9afb6b8bb